lockbox-ios/View/ItemDetailView.swift

```diff
@@ -202,3 +209,20 @@ extension ItemDetailView: UIGestureRecognizerDelegate {
         }
     }
 }
+
+extension ItemDetailView: UITableViewDragDelegate {
+    func tableView(_ tableView: UITableView, itemsForBeginning session: UIDragSession, at indexPath: IndexPath) -> [UIDragItem] {
+        let cell = tableView.cellForRow(at: indexPath)
+        let data = (cell as? ItemDetailCell)?.dragValue?.data(using: .utf8)
+        let itemProvider = NSItemProvider()
+
+        itemProvider.registerDataRepresentation(forTypeIdentifier: kUTTypePlainText as String, visibility: .all) { completion in
+            completion(data, nil)
```

Comment:
Can this callback also record a copy(Username/Password) telemetry event?  Or is this not the right place?

Reply:
Also, we really want to `touch()` the entry, to update its `timeLastUsed` and `timesUsed`.